Jenkinsfile

```diff
@@ -368,6 +374,26 @@ pipeline {
   }
 }
 
+def uploadToCDN() {
+  def source = 'packages/rum/dist/bundles/*.js'
+  def target = 'gs://apm-rum-357700bc'
+  def secret = 'secret/gce/elastic-cdn/service-account/apm-rum-admin'
+  def version = sh(label: 'Get package version', script: 'jq --raw-output .version packages/rum/package.json', returnStdout: true)
+  def majorVersion = "${version.split('\\.')[0]}.x"
+
+  // Publish version with the semver format and cache them for 1 year.
+  publishToCDN(headers: ["Cache-Control:public,max-age=31536000,immutable"],
```

Comment:
one question on the path,  is the CDN path configured from `<hostname>/bundles` -> GCP bucket or `<hostname>/` - > GCP bucket. 

I am confused as i am not able to see `bundles` in the target path.

Reply:
Let me ask infra

Reply:
As far as I understood they just copy what we put in the folder, so, I think we don't need `bundles`